app/build.gradle.kts

```diff
@@ -94,7 +94,7 @@ dependencies {
 	testImplementation(libs.test.guava)
 	testImplementation(libs.test.kotest.datatest)
 	testImplementation(libs.test.mockk)
-	testCompileOnly("org.jetbrains.kotlin:kotlin-reflect:1.9.20") {
+	testCompileOnly("org.jetbrains.kotlin:kotlin-reflect:1.9.10") {
```

Comment:
Fixes:
```
Could not resolve org.jetbrains.kotlin:kotlin-reflect:1.9.20.
     Required by:
         project :app
      > Cannot find a version of 'org.jetbrains.kotlin:kotlin-reflect' that satisfies the version constraints:
           Dependency path 'Picover:app:unspecified' --> 'org.jetbrains.kotlin:kotlin-reflect:1.9.20' because of the following reason: Needed to locally trigger single kotest test - check new versions of kotlin and kotest plugins to fix this workaround
           Constraint path 'Picover:app:unspecified' --> 'org.jetbrains.kotlin:kotlin-reflect:***strictly 1.9.10***' because of the following reason: debugUnitTestRuntimeClasspath uses version 1.9.10

   > Could not resolve org.jetbrains.kotlin:kotlin-reflect:***strictly 1.9.10***.
     Required by:
         project :app
      > Cannot find a version of 'org.jetbrains.kotlin:kotlin-reflect' that satisfies the version constraints:
           Dependency path 'Picover:app:unspecified' --> 'org.jetbrains.kotlin:kotlin-reflect:1.9.20' because of the following reason: Needed to locally trigger single kotest test - check new versions of kotlin and kotest plugins to fix this workaround
           Constraint path 'Picover:app:unspecified' --> 'org.jetbrains.kotlin:kotlin-reflect:***strictly 1.9.10***' because of the following reason: debugUnitTestRuntimeClasspath uses version 1.9.10